src/clojure_sandbox/4clojure.clj

```diff
@@ -0,0 +1,337 @@
+(ns clojure-sandbox.4clojure
+  (:require [clojure.string :as str]))
+
+;; 21. Write a function which returns the Nth element from a sequence.
+(defn nth-element [[current & remaining :as items] position]
+  (when items
+    (if (pos? position)
+      (recur remaining (dec position))
+      current)))
+
+(nth-element [] 1)
+(nth-element [nil 1 2 3 4] 2)
+(nth-element [0 1 2 3] 5)
+
+;; 22. Write a function which returns the total number of elements in a sequence.
+(defn count-total [init-sequence]
```

Comment:
Slightly subjective, but you could drop the init from the name. The fact that it's the _init_ sequence is only relevant in the context of the implementation which does tail recursion with other sequences.

It's not pertinent to the API that it is providing.